src/main/java/hydrator/implementation/RandomValueHydratorFactory.java

```diff
@@ -0,0 +1,64 @@
+package hydrator.implementation;
+
+import graphql.schema.GraphQLEnumValueDefinition;
+import hydrator.IEnumHydrator;
+import hydrator.IScalarHydrator;
+import hydrator.IValueHydratorFactory;
+import org.fluttercode.datafactory.impl.DataFactory;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * Implementation of {@link IValueHydratorFactory} that returns random data.
+ */
+public class RandomValueHydratorFactory implements IValueHydratorFactory {
+    private final IScalarHydrator scalarHydrator;
+    private final IEnumHydrator enumHydrator;
+
+    public RandomValueHydratorFactory() {
+        this.scalarHydrator = new RandomScalarHydrator();
+        this.enumHydrator = new RandomEnumHydrator();
+    }
+
+    @Override
+    public IScalarHydrator provideScalarHydrator() {
+        return scalarHydrator;
+    }
+
+    @Override
+    public IEnumHydrator provideEnumHydrator() {
+        return enumHydrator;
+    }
+
+    /*******************************************************************************************************************
+     * RANDOM VALUE HYDRATORS
+     ******************************************************************************************************************/
+
+    private static class RandomScalarHydrator implements IScalarHydrator {
```

Comment:
I'm surprised to see we're not using `DataFetchers` for these, any strong reason why we're doing this manually?

Reply:
DataFetchers require us to be part of the graphql-java lifecycle. However, even if we did use them, we would probably need a strategy on how to generate the data returned, which would be handled by the class above.